scss/_patterns_buttons.scss

```diff
@@ -117,17 +174,63 @@ $hover-background-opacity-percentage: $hover-background-opacity-amount * 100%;
     @include vf-focus($color-focus-negative);
   }
 
+  %p-button--negative-dark {
+    $color-negative-dark: adjust-color($color-negative, $lightness: -10%);
+
+    @include vf-button-pattern(
+      $button-background-color: $color-negative-dark,
+      $button-hover-background-color: adjust-color($color-negative-dark, $lightness: -$hover-background-opacity-percentage),
+      $button-active-background-color: adjust-color($color-negative-dark, $lightness: -$active-background-opacity-percentage),
+      $button-disabled-background-color: $color-negative-dark,
+      $button-border-color: $color-negative-dark,
+      $button-hover-border-color: adjust-color($color-negative-dark, $lightness: -$hover-background-opacity-percentage),
+      $button-active-border-color: adjust-color($color-negative-dark, $lightness: -$active-background-opacity-percentage),
+      $button-disabled-border-color: $color-negative-dark,
+      $button-text-color: vf-contrast-text-color($color-negative-dark)
+    );
+
+    @include vf-focus($color-focus-negative);
+  }
+
   .p-button--negative {
-    @extend %p-button--negative;
+    @extend %vf-button-base;
+
+    // Theming
+    @if ($theme-default-p-button == 'dark') {
+      @extend %p-button--negative-dark;
+
+      &.is-light {
+        @extend %p-button--negative-light;
+      }
+    } @else {
+      @extend %p-button--negative-light;
+
+      &.is-dark {
+        @extend %p-button--negative-dark;
+      }
+    }
+
     @extend %vf-button-white-success-icon;
   }
 }
 
 @mixin vf-button-base {
-  %p-button--base {
-    @extend %vf-button-base;
+  %p-button--base-light {
```

Comment:
@lyubomir-popov Now with dark/light themes we should be able to change background on base button back to transparent?